index.d.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+export interface TruncateProps {
+    lines?: number;
+    ellipsis?: React.ReactNode | ((element: React.ReactNode) => React.ReactNode);
```

Comment:
this could be `string`, `ReactNode` or `function`. looks like `string` is not covered here?

Reply:
ReactNode covers string

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L148